datasource_service/tests/test_main.py

```diff
@@ -0,0 +1,21 @@
+""" Pytest for the GET api { Mock the nexrad object so that it doesn't download the radar object for the
+unit testing } """
+
+from fastapi.testclient import TestClient
+
+from datasource_service.main import app
+
+test_client = TestClient(app)
+
+
```

Comment:
Looks like a test case is missing for the function: download_radar_object. Could you please add some for the same. It will be great to debug then.